packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -167,11 +169,10 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
             emit AssertionSettled(assertionId, assertion.proposer, false, true);
         } else {
             // Dispute, settle with the disputer
-            int256 dvmResolvedPrice =
+            int256 resolvedPrice =
                 _getOracle(assertionId).getPrice(identifier, assertion.assertionTime, _stampAssertion(assertionId)); // Revert if price not resolved.
 
-            assertion.settlementResolution = dvmResolvedPrice == 1e18;
-            // todo: if (assertion.useDisputeResolution)
```

Comment:
No real need to handle it here. Its was easier just to return false when assertion result is requested.